homeassistant/components/todoist/calendar.py

```diff
@@ -216,11 +219,12 @@ def handle_new_task(call):
         if PRIORITY in call.data:
             item.update(priority=call.data[PRIORITY])
 
```

Comment:
This is the primary change to get things working with the latest version.  The `due` parameter is [now an object](https://developer.todoist.com/sync/v8/?python#due-dates)